socorro/processor/cache_manager.py

```diff
@@ -449,14 +479,24 @@ def _event_generator(self, nonblocking=False):
                         len(self.lru),
                         f"{self.total_size:,d}",
                     )
-                    # Emit a usage metric, but debounce it so it only gets emitted at
-                    # most once per second
-                    now = datetime.datetime.now()
-                    if now > (last_usage_update + datetime.timedelta(seconds=1)):
```

Comment:
It's weird that I was creating datetimes and timedeltas to calculate this rather than using floats. What was I thinking?